django-backend/fecfiler/settings/local.py

```diff
@@ -22,6 +22,11 @@
 # E2E Testing Login API
 E2E_TESTING_LOGIN = True
 
+MIDDLEWARE.append(  # NOSONAR # noqa F405
+    "django_cprofile_middleware.middleware.ProfilerMiddleware"
+)
+DJANGO_CPROFILE_MIDDLEWARE_REQUIRE_STAFF = False
+
```

Comment:
This adds a performance profiling tool that is *only* included on local.  I think it's worth keeping, but it's optional.

Reply:
Let's move the addition of the middleware into a new ticket so we can integrate it into the Docker image we use for testing, Dockerfile-e2e. We can also move the declaration of it into the requirements-test.txt config file to further isolate it into testing setups. I'll write up a new ticket.

Reply:
Done :+1: